### PR TITLE
Fix: on android, Device::setAccelerometerInterval will causes dead loop

### PR DESCRIPTION
**Pull request by xhcnb**
it's a low level mistake  
: )

### CODE REVIEW

File: cocos/2d/platform/android/CCDevice.cpp [hunk 1]
```diff
@@ -28,7 +28,7 @@ void Device::setAccelerometerEnabled(bool isEnabled)
 
 void Device::setAccelerometerInterval(float interval)
 {
-	setAccelerometerInterval(interval);
+	::setAccelerometerInterval(interval);
```
Comment (minggo) [inline cocos/2d/platform/android/CCDevice.cpp]: Thanks.
But it think it is better to change the jni function name.

**Comment by xhcnb** [comment]
@minggo ,U are right, Just renamed the native functions.

**Comment by xhcnb** [comment]
Renamed.
Thanks for your guys hard working, hope for 3.0 release.

**Comment by minggo** [comment]
Thanks.

**Pull request merged**